wellregistry/registry/admin/monitoring_location.py

```diff
@@ -281,7 +375,7 @@ class SiteNoFilter(AutocompleteFilter):
     field_name = 'site_no'
     use_pk_exact = False
     parameter_name = 'site_no__exact'
-    template="admin/site_no-filter.html"
+    template = "admin/site_no-filter.html"
```

Comment:
interesting, when screensharing, I presumed the IDE removed the spaces around the '=' but it looks like it didn't

Reply:
fixed